src/Apps/Order/Routes/Shipping2/__tests__/FulfillmentDetailsForm.jest.tsx

```diff
@@ -55,6 +56,31 @@ const renderTree = testProps => {
   return { wrapper }
 }
 
+let initialValues: ShipValues = {
```

Comment:
Is this ever reassigned?

Reply:
Not yet, I guess i should make it a const.